cloud_experts_tutorials/cloud-experts-external-dns.adoc

```diff
@@ -65,14 +72,14 @@ ROSA manages secondary Ingress Controllers using the Custom Domain Operator. The
 
 .Procedure
 
-. Create a new project
+. Create a new project:
 +
 [source,terminal]
 ----
 $ oc new-project external-dns-operator
 ----
 
-. Create a new TLS secret from a private key and a public certificate, where `fullchain.pem` is your full wildcard certificate chain (including any intermediaries) and `privkey.pem` is your wildcard certificate's private key.
+. Create a new TLS secret from a private key and a public certificate, where `fullchain.pem` is your full wildcard certificate chain (including any intermediaries) and `privkey.pem` is your wildcard certificate's private key:
 +
 .Example
```

Comment:
Is this an example, or the command that would need to be used to get the desired output? I don't see 'Example' used often in this way, but I could be totally wrong, and don't know enough about the specific steps to say. :(